src/network/networkprotocol.h

```diff
@@ -937,6 +938,8 @@ enum AccessDeniedCode {
 	SERVER_ACCESSDENIED_ALREADY_CONNECTED,
 	SERVER_ACCESSDENIED_SERVER_FAIL,
 	SERVER_ACCESSDENIED_CUSTOM_STRING,
+	SERVER_ACCESSDENIED_SHUTDOWN,
+	SERVER_ACCESSDENIED_CRASH,
```

Comment:
I realize this is kind of nitpicky, but the word "crash" does not exactly instil a lot of confidence in users, even if it appears in the source code.  Could you change it to something like SERVER_ACCSESDENIED_INTERNAL_ERROR?